src/build_system.ml

```diff
@@ -736,7 +736,7 @@ end = struct
               String.Map.set aliases "default"
                 { deps = Path.Set.empty
                 ; dyn_deps =
-                    (Alias0.dep_rec_internal ~name:"install" ~dir ~ctx_dir
+                    (Alias0.dep_rec_internal ~name:"all" ~dir ~ctx_dir
```

Comment:
This should be `all` in dune >=2 projects, and `install` in other projects. Otherwise, the interpretation of `(alias xxx/default)` would depend on the version of the dune binary rather than the version of the dune language used in the project.

Reply:
Yeah, that makes sense. Unfortunately, it's not so easy to do this. The build_system would have to be aware of the dune_version for every single directory. That seems quite excessive.

Reply:
Isn't it already available in the file tree? :)